Web/Case Study Assignment-5/CaseStudyAssignment-5/WebContent/scripts/scriptMain.js

```diff
@@ -0,0 +1,8 @@
+// JavaScript Document
+window.onload = function() {
```

Comment:
ACTION: There is no need to create a new js file for single method and you are using this js file in single place.